.travis.yml

```diff
@@ -4,17 +4,34 @@ sudo: false
 cache:
   - pip
 
+before_cache:
+  - rm ~/.cache/pip/log/debug.log
+
 language: python
 python:
   - "2.7"
   - "3.5"
 
 install:
-  - pip install -r requirements.txt
   - pip install tox-travis
 
 script:
-  # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
-  - ansible-playbook --syntax-check playbooks/byo/config.yml
   - tox
-  - cd utils && tox
+
+notifications:
+  email:
+    recipients:
+      - jdetiber@redhat.com
+      - sdodson@redhat.com
```

Comment:
We have a few other options to prevent it, but they all have drawbacks.

We could force PRs to be up to date with HEAD prior to merge, but that would cause everyone a lot of pain and frustration.

Forcing rebase merges might help as well, but I worry it would cause other potential issues.

With the CI/CD work, I think leveraging that for a merge queue (when ready) is the least painful approach.